### PR TITLE
#1066 api ml standalone

### PR DESCRIPTION
**Pull request by janan07**
This Pull Request addresses issue [#1066](https://github.com/zowe/api-layer/issues/1066) in api-layer.

### CODE REVIEW

**Review by jalel01 — APPROVED** [review]

**Review by Joe-Winchester — APPROVED** [review]
If LAUNCH_COMPONENT_GROUPS is GATEWAY without ZSS being launched, then does this impede any behavior such as client side certificate authentication ?

File: docs/user-guide/api-mediation-standalone.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+ Services are not required, it is possible to specify which components start in the
+ context of Zowe. No change is required during the installation process to
+ support this setup.
+ 
+Once Zowe is installed, use the following procedure to limit which components start.
+
+**Follow these steps:**
+
+1. Open the file `<Zowe instance directory>/instance.env`.
+2. Find the property `LAUNCH_COMPONENT_GROUPS` and set `GATEWAY`
```
Comment (stevenhorsman) [inline docs/user-guide/api-mediation-standalone.md:15]: This doesn't result in just the APIML starting. The GATEWAY component group includes the explorer UI and API components.
If you want to just run the APIML then you can set `ZWE_LAUNCH_COMPONENTS=discovery,gateway,api-catalog` (and caching-service if/when required). This was added for HA support and isn't in any of the automated testing, so probably worth manually checking it's still okay given the large number of refactors going on in that area

**Review by stevenhorsman — CHANGES_REQUESTED** [review]

**Comment by balhar-jakub** [comment]
@Joe-Winchester Yeah, the x509 client certificates won't work without the ZSS properly set and run.

File: docs/user-guide/api-mediation-standalone.md (hunk 1 above)
Comment (janan07) [reply docs/user-guide/api-mediation-standalone.md]: The property `LAUNCH_COMPONENT_GROUPS`  set to  `GATEWAY` has been replaced with 
 property `ZWE_LAUNCH_COMPONENTS` and set to `discovery,gateway,api-catalog`

**Review by stevenhorsman — APPROVED** [review]

**Review by balhar-jakub — APPROVED** [review]

**Pull request merged**